index.js

```diff
@@ -4,20 +4,23 @@ exports.byteLength = byteLength
 exports.toByteArray = toByteArray
 exports.fromByteArray = fromByteArray
 
-var lookup = []
-var revLookup = []
+var lookups = { _: [], url: [] }
+var revLookups = { _: [], url: null }
```

Comment:
Any reason this is named `_`? I feel like `'default'` would be clearer.

Reply:
i was trying to avoid using a reserved keyword but it's probably not an issue here. i added a commit to change this

Reply:
Cool, thanks!